.changeset/loud-meals-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sl-design-system/tokens": patch
```

Comment:
The fix is connected with a `teas` package so here instead of `@sl-design-system/tokens`
we should have `@sl-design-system/teas` :)